adapters/appnexus/appnexus.go

```diff
@@ -486,6 +488,9 @@ func getMediaTypeForBid(bid *appnexusBidExt) (openrtb_ext.BidType, error) {
 // getIabCategoryForBid maps an appnexus brand id to an IAB category.
 func (a *AppNexusAdapter) getIabCategoryForBid(bid *appnexusBidExt) (string, error) {
 	// TODO: Change from BrandId to a CategoryId once that is returned from impbus
+	if &bid.Appnexus == nil || bid.Appnexus.BrandId == 0 {
```

Comment:
Should be `if bid == nil || bid.Appnexus.BrandId == 0 {` as long as bid valid Appnexus field will always exist even if it's initialized with all zero values.

Reply:
Done